status-network-contracts/test/stakemanager/IntegrationTest.t.sol

```diff
@@ -0,0 +1,369 @@
+pragma solidity ^0.8.26;
+
+import { StakeManagerTest } from "./StakeManagerBase.t.sol";
+
+contract IntegrationTest is StakeManagerTest {
+    function setUp() public virtual override {
+        super.setUp();
+    }
+
+    function testStakeFoo() public {
```

Comment:
I think now is a good time to rename this to something more meaningful.
I renamed it to this a long time ago while testing something and left it unchanged after.